groups/models.py

```diff
@@ -161,6 +161,20 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def subscribers(self):
```

Comment:
Views have a tendency to be a dumping ground of code. In this case the method only needs to know information stored on the `BaseComment` model - the view shouldn't need to be aware of these implementation details.